docs/src/pages/basics/guide-angular/index.md

```diff
@@ -0,0 +1,116 @@
+* * *
+
+id: 'guide-angualr'
+
+## title: 'Storybook for Angular'
+
+You may have tried to use our quick start guide to setup your project for Storybook. If you want to set up Storybook manually, this is the guide for you.
+
+> This will also help you understand how Storybook works.
+
+## Starter Guide Angular
+
+Storybook has its own Webpack setup and a dev server.  
```

Comment:
An initial letter should be a lowercase letter;)
see: https://github.com/webpack/media#name